packages/cf.js/considerations-for-0-0-2.md

```diff
@@ -0,0 +1,13 @@
+# missing features of v0.0.2
+
+- need a way to provide a `timeout` during the install flow
+    - argument to `proposeInstall`?
+    - attribute of the manifest.json
+- currently, the user's address is never being set
+    - provide as an argument: `new Client(nodeProvider, address)`
+    - fetch it from the wallet through an `async init` method?
+- do dapp's specify the deposit? if so, `proposeInstall` and `install` need a deposit arg
```

Comment:
I'm still not clear on this. Doesn't the proposeInstall step need to specify _both_ deposits? Isn't the app expected to handle terms negotiation?

Reply:
My understanding of `proposeInstall` is that it is the initial contact between parties, and so the initiator wouldn't yet have the counterparty's deposit amount. This would be an optimal time for them to provide their own deposit amount though, and then the counterparty would be able to provide both their own deposit amount and the initiator's deposit to `install`.

Reply:
Okay. The initiator would still get to Accept or Decline the `install` in their node, right?

Reply:
I believe so. Can you confirm, @snario?

Reply:
Yes.

That's part of the validation logic.